image/base_image_manager.go

```diff
@@ -42,6 +42,20 @@ func (bim BaseImageManager) syncImageLocal(image v1.Image) (err error) {
 	return nil
 }
 
```

Comment:
BaseImageManager aims to provide common abilities. I support implement deleteImageLocal within defaultImageService is good. Or implement a imageStore like layerStore(not urgent)

Reply:
The baseimageManager's implementation is just like a imageStore. So I consider providing deleteImageLocal in BaseImageManager is reasonable.

Reply:
agree